apps/web/pages/api/social/og/image.tsx

```diff
@@ -108,8 +110,8 @@ export default async function handler(req: NextApiRequest) {
         description: searchParams.get("description"),
         imageType,
       });
-      const description_ = await (await remark().use(strip).process(description)).toString();
-      const img = new ImageResponse(<Generic title={title} description={description_} />, ogConfig) as {
+
+      const img = new ImageResponse(<Generic title={title} description={description} />, ogConfig) as {
```

Comment:
Is this expected?

Reply:
It is expected.. On my last PR to fix this problem I was cleaning up on the wrong template 😫, so I reverted back to how it was .